Docs/data_providers.md

```diff
@@ -0,0 +1 @@
+
```

Comment:
Nit: let's not include empty files, it'd be a bit confusing for the reader.

Reply:
done! removed empty files